js/src/hooks/useScript.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { useEffect } from '@wordpress/element';
+
+/**
+ * Load an external script and call the provided callback function when the script is loaded.
+ *
+ * @param {string}   url      The script's url.
+ * @param {Function} callback A function to be called after the script is loaded.
+ */
+const useScript = ( url, callback = () => {} ) => {
```

Comment:
💅  The default value of `callback` is not needed if it's a required parameter.

Reply:
Thanks! Updated in c426ee5.